examples/evm/call-contract-with-token-express/index.js

```diff
@@ -1,112 +0,0 @@
-'use strict';
```

Comment:
Why did we remove the express example? Don't we need this one? I thought Euro built it so that this worked online with an existing testnet contract.

Reply:
sorry I had taken out express, but re-adding it now as I referenced it in the other pr for the express doc